docs/content.zh/docs/concepts/architecture.md

```diff
@@ -24,57 +24,66 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-# Architecture
+<a name="architecture"></a>
 
-Flink Kubernetes Operator (Operator) acts as a control plane to manage the complete deployment lifecycle of Apache Flink applications. The Operator can be installed on a Kubernetes cluster using [Helm](https://helm.sh). In most production environments it is typically deployed in a designated namespace and controls Flink deployments in one or more managed namespaces. The custom resource definition (CRD) that describes the schema of a `FlinkDeployment` is a cluster wide resource. For a CRD, the declaration must be registered before any resources of that CRDs kind(s) can be used, and the registration process sometimes takes a few seconds.
+# 架构
 
-{{< img src="/img/concepts/architecture.svg" alt="Flink Kubernetes Operator Architecture" >}}
-> Note: There is no support at this time for [upgrading or deleting CRDs using Helm](https://helm.sh/docs/chart_best_practices/custom_resource_definitions/).
+Flink Kubernetes Operator（Operator）充当控制平面，用于管理 Apache Flink 应用程序的完整 deployment 生命周期。可以使用 [Helm](https://helm.sh) 在 Kubernetes 集群上安装 Operator。在大多数生产环境中，它通常部署在指定的命名空间中，并控制一个或多个 Flink 部署到受托管的 namespaces 。描述 `FlinkDeployment` 模式的自定义资源定义（CRD）是一个集群范围的资源。对于 CRD，必须在使用该 CRD 类型的任何资源之前注册声明，注册过程有时需要几秒钟。
```

Comment:
1. `Flink 部署` -> `Flink deployments `
2. How about translate `schema` to `结构` instead of `模式`?
3. As https://cwiki.apache.org/confluence/display/FLINK/Flink+Translation+Specifications mentioned, `是一个集群范围的资源`, we don't need `一个` here.

<img width="978" alt="image" src="https://github.com/apache/flink-kubernetes-operator/assets/38427477/4b5153c8-73b1-4708-9bb4-83255f7d29a6">

Reply:
Sorry, I overlooked this, next time I should read it aloud to check